core/lib/network-request.js

```diff
@@ -80,14 +80,10 @@ const HEADER_PROTOCOL_IS_H2 = 'X-ProtocolIsH2';
 
 /**
```

Comment:
clarified these docstrings, and formatted such that they apply to the correct variables (it was off...)